### ISSUE TITLE
strange bullet points / missing text / layout problem

### ISSUE THREAD

**Issue by alicetragedy** [vienna-rb/vienna-rb.github.com#10]
just noticed that there are two bullet points next to each other at the bottom of each blog post, between "posted by" and the post category. Looks as though there's supposed to be some text there (maybe a post date is missing?)

![screen shot 2015-01-15 at 12 03 51](https://cloud.githubusercontent.com/assets/3143348/5756364/ffe38132-9cae-11e4-9d62-04b81e3de3c1.png)

**Comment by geraldb** [comment]
@alicetragedy   Good point. If you're interested I can try to fix the list issue in the (new) site (that tries to keep the same look 'n' feel but uses a new (simple) all-in-one single css file e.g. style.scss
